frontend/apps/web/components/TruncatedText.tsx

```diff
@@ -7,12 +7,15 @@ import {
   TooltipTrigger,
 } from './ui/tooltip';
 
+type TooltipSide = 'top' | 'right' | 'bottom' | 'left';
+type TooltipAlign = 'start' | 'center' | 'end';
 interface Props {
   text: string;
-
+  side?: TooltipSide;
+  align?: TooltipAlign;
```

Comment:
Instead of creating your own TS types, you can just use the ones provided by the tooltipcontent props.
```suggestion
  side?: TooltipContentProps['side']
  align?: TooltipContentProps['align']
```